packages/editor/src/package/serlo-renderer.tsx

```diff
@@ -0,0 +1,29 @@
+import { StaticRenderer } from '@editor/static-renderer/static-renderer'
+import type { AnyEditorDocument } from '@editor/types/editor-plugins'
+
+import { InstanceDataProvider } from '@/contexts/instance-context'
+import { LoggedInDataProvider } from '@/contexts/logged-in-data-context'
+import type { InstanceData, LoggedInData } from '@/data-types'
+
+export interface SerloRendererProps {
+  instanceData: InstanceData
+  loggedInData: LoggedInData
+  documentState: AnyEditorDocument | AnyEditorDocument[]
```

Comment:
calling it `documentState` complicates things below. 
I'd just go with `document` (since state can mean so many things it's basically meaningless :).
But you can also merge it as it is.

Reply:
Good point! I like `document` more as well. Changed!